packages/web-app-files/src/views/SharedWithMe.vue

```diff
@@ -256,3 +457,28 @@ export default {
   }
 }
 </script>
+
+<style>
+.centered {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+.shares-bar {
+  display: flex;
+  flex-direction: row;
+  align-items: baseline;
+}
+
+#pending-highlight {
+  background-color: aliceblue;
+}
+.show-hide-pending {
+  text-align: center;
+  padding-bottom: 12px;
+}
+
+.margin-div {
+  margin-left: 24px;
```

Comment:
This class is not needed. You can use our margin classes. `oc-ml` which uses the medium margin for `**m**argin **l*eft`.